docs/developer_guide/playbooks.md

```diff
@@ -205,3 +205,18 @@ $ docker run --rm -it \
     ansible-playbook create_object.yml -e "customer=patient0 env_type=staging" \
         -e "object_template=templates/openshift/edxapp/job/import_demo_course.yml.j2"
 ```
+
+## `create_vaults.yml`
+
+This playbook will be used to create the vault files for a given application. It will reads the databases vault
+(must be present in `groups_vars/customers/{customer}/{env_type}/secrets/databases.ynl.vault`) and creates the vault
+application based on its template (`app/{app.name}/vars/vault/main.yml.j2`).
+This playbook will not work on all the applications present in your `apps` folder. Runs it only with a specific app.
```

Comment:
Suggestion:

---

This playbook generates Ansible vaults (required to run applications) from templates bundled with the applications themselves (see `apps/{{ app.name }}/vars/vault/main.yml.j2`).

> Note that this playbook requires the `databases.vault.yml` vault as input to generate applications vaults. We expect this vault to be in `groups_vars/customers/{{ customer }}/{{ env_type }}/secrets/databases.vault.yml`.